dags/atd_visionzero_cr3_process_missing_pdfs.py

```diff
@@ -0,0 +1,84 @@
+import os
+
+from airflow.models import DAG
+from airflow.operators.docker_operator import DockerOperator
+from pendulum import datetime, duration
+
+from utils.onepassword import get_env_vars_task
+from utils.knack import get_date_filter_arg
+from utils.slack_operator import task_fail_slack_alert
+
+DEPLOYMENT_ENVIRONMENT = os.getenv("ENVIRONMENT", "development")
+
+DEFAULT_ARGS = {
+    "owner": "airflow",
+    "depends_on_past": False,
+    "start_date": datetime(2015, 1, 1, tz="America/Chicago"),
+    "email_on_failure": False,
+    "email_on_retry": False,
+    "retries": 0,
+    "retry_delay": duration(minutes=5),
+    "on_failure_callback": task_fail_slack_alert,
+}
+
+# secrets atd_visionzero_hasura_sql_production
+
+REQUIRED_SECRETS = {
+    "HASURA_ENDPOINT": {
+        "opitem": "Vision Zero graphql-engine Endpoints",
+        "opfield": f"{DEPLOYMENT_ENVIRONMENT}.Query Endpoint",
+    },
+    "HASURA_ADMIN_KEY": {
+        "opitem": "Vision Zero graphql-engine Endpoints",
+        "opfield": f"{DEPLOYMENT_ENVIRONMENT}.Admin Key",
+    },
+    "AWS_BUCKET_NAME": {
+        "opitem": "CRIS CR3 Download",
+        "opfield": f"{DEPLOYMENT_ENVIRONMENT}.AWS_CRIS_CR3_BUCKET_NAME",
+    },
+    "AWS_BUCKET_ENVIRONMENT": {
+        "opitem": "CRIS CR3 Download",
+        "opfield": f"{DEPLOYMENT_ENVIRONMENT}.AWS_CRIS_CR3_BUCKET_PATH_ENVIRONMENT",
+    },
+}
+
+with DAG(
+    dag_id="vision_zero_process_missing_pdfs",
```

Comment:
fwiw our other Vision Zero DAGs follow the naming convention `vz_blah_blah` instead of `vision_zero_blah_blah`

Reply:
Thank you for this observation. Great catch - I've unified the naming scheme.